### PR TITLE
feat: introduce /construction_project retrieve view

### PR DESCRIPTION
**Pull request by open-dynaMIX**

### CODE REVIEW

File: ebau_gwr/core/hs_client.py [hunk 1]
```diff
@@ -47,8 +72,16 @@ def get(self, path="constructionprojects", query=None, headers=None):
...
 
-        data = CreateFromDocument(content)
+        try:
+            data = CreateFromDocument(content)
+        except (
+            IncompleteElementContentError,
+            UnprocessedElementContentError,
+        ) as e:  # pragma: no cover
+            # TODO: correctly report errors back
+            print(e.details())
```
Comment (winged) [inline ebau_gwr/core/hs_client.py:85]: Maybe use log instead of print?

File: ebau_gwr/core/serializers.py [hunk 2]
```diff
@@ -1,9 +1,93 @@
 from rest_framework import serializers
 
 
+class DateField(serializers.DateField):
```
Comment (winged) [inline ebau_gwr/core/serializers.py:4]: What's the difference to the parent class? Why is this needed?

File: ebau_gwr/core/tests/ech_mock_data.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 getConstructionProjectResponse = (
     b'<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n'
-    b'<ns2:response xmlns="http://www.ech.ch/xmlns/eCH-0058/5" xmlns:ns2="http://www.ech.ch/xmlns/eCH-0216/2" xmlns:ns3="http://www.ech.ch/xmlns/eCH-0129/5" xmlns:ns4="http://www.ech.ch/xmlns/eCH-0044/4" xmlns:ns5="http://www.ech.ch/xmlns/eCH-0097/2" xmlns:ns6="http://www.ech.ch/xmlns/eCH-0010/6" xmlns:ns7="http://www.ech.ch/xmlns/eCH-0007/6" xmlns:ns8="http://www.ech.ch/xmlns/eCH-0008/3">\n'
+    b'<ns2:response xmlns="http://www.ech.ch/xmlns/eCH-0058/5" xmlns:ns2="http://www.ech.ch/xmlns/eCH-0216/1" xmlns:ns3="http://www.ech.ch/xmlns/eCH-0129/5" xmlns:ns4="http://www.ech.ch/xmlns/eCH-0044/4" xmlns:ns5="http://www.ech.ch/xmlns/eCH-0097/2" xmlns:ns6="http://www.ech.ch/xmlns/eCH-0010/6" xmlns:ns7="http://www.ech.ch/xmlns/eCH-0007/6" xmlns:ns8="http://www.ech.ch/xmlns/eCH-0008/3">\n'
```
Comment (winged) [inline ebau_gwr/core/tests/ech_mock_data.py:3]: Hmm, it might be a better idea to put this into .xml files instead, and read those

**Review by winged — COMMENTED** [review]
Nice work! Just some open questions & suggestions (may or may not require change)

File: ebau_gwr/core/hs_client.py (hunk 1 above)
Comment (open-dynaMIX) [reply ebau_gwr/core/hs_client.py]: Done. But it's only for development until we have proper error handling.

File: ebau_gwr/core/serializers.py (hunk 2 above)
Comment (winged) [reply ebau_gwr/core/serializers.py:4]: Hmm. Might be useful to name it `PyxbDateField` then, to avoid confusion

Comment (open-dynaMIX) [reply ebau_gwr/core/serializers.py]: Done.

File: ebau_gwr/core/tests/ech_mock_data.py (hunk 3 above)
Comment (open-dynaMIX) [reply ebau_gwr/core/tests/ech_mock_data.py]: Done.

**Review by winged — APPROVED** [review]
:100:
